src/lib/structures/fpa_rounding.ml

```diff
@@ -93,8 +93,16 @@ let fpa_rounding_mode_dty, d_cstrs, fpa_rounding_mode =
     List.map (fun c -> DStd.Path.global @@ to_smt_string c, []) cstrs
   in
   let _, d_cstrs = DE.Term.define_adt ty_cst [] cstrs in
+  let () =
+    match DStd.Expr.Ty.definition ty_cst with
+    | Some def ->
+      let name, hash = Nest.generate [def] |> List.hd in
```

Comment:
This should not use `List.hd` (which fails if the list is empty but might silently introduce errors if it has more than one element) but rather an explicit pattern matching.

Reply:
I removed this code.